5.0/fr/0x15-V6-Authentication.md

```diff
@@ -0,0 +1,166 @@
+
+| # | Description | Niveau |
+| :---: | :--- | :---: |
+| **6.4.1** | Vérifiez que les mots de passe initiaux ou les codes d'activation générés par le système sont générés de manière aléatoire et sécurisée, qu'ils respectent la politique de mots de passe en vigueur et qu'ils expirent après une courte période ou après leur première utilisation. Ces secrets initiaux ne doivent pas devenir des mots de passe permanents. | 1 |
+| **6.4.2** | Vérifiez que les indices de mot de passe ou l'authentification basée sur les connaissances (appelées « questions secrètes ») ne sont pas présents. | 1 |
+| **6.4.3** | Vérifiez qu'un processus sécurisé de réinitialisation d'un mot de passe oublié est mis en œuvre, qui ne contourne aucun mécanisme d'authentification multifacteur activé. | 2 |
+| **6.4.4** | Vérifiez que si un facteur d'authentification multifacteur est perdu, la preuve de vérification d'identité est effectuée au même niveau que lors de l'inscription. | 2 |
+| **6.4.5** | Vérifiez que les instructions de renouvellement des mécanismes d'authentification qui expirent sont envoyées avec suffisamment de temps pour être exécutées avant l'expiration de l'ancien mécanisme d'authentification, en configurant des rappels automatiques si nécessaire. | 3 |
+| **6.4.6** | Vérifiez que les administrateurs peuvent initier la réinitialisation du mot de passe de l'utilisateur, mais que cela ne leur permet pas de modifier ou de choisir son mot de passe. Cela évite qu'ils ne connaissent le mot de passe de l'utilisateur. | 3 |
+
+## V6.5 Exigences générales en matière d'authentification multifacteur
+
+Cette section fournit des conseils généraux qui seront pertinents pour différentes méthodes d’authentification multifactorielle.
+
+Les mécanismes comprennent :
+
+* Recherche de secrets
+* Mots de passe à usage unique (TOTP)
+* Mécanismes hors bande
+
+Les secrets de recherche sont des listes pré-générées de codes secrets, similaires aux numéros d'autorisation de transaction (TAN), aux codes de récupération des réseaux sociaux ou à une grille contenant un ensemble de valeurs aléatoires. Ce type de mécanisme d'authentification est considéré comme « quelque chose que vous possédez », car les codes sont volontairement non mémorisables et doivent donc être stockés quelque part.
+
+Les mots de passe à usage unique temporaires (TOTP) sont des jetons physiques ou logiciels qui affichent un code pseudo-aléatoire à usage unique et changeant en permanence. Ce type de mécanisme d'authentification est considéré comme « quelque chose que vous possédez ». Les TOTP multifactoriels sont similaires aux TOTP monofactoriels, mais nécessitent la saisie d'un code PIN valide, d'un déverrouillage biométrique, d'une clé USB ou d'un appairage NFC, ou d'une valeur supplémentaire (comme des calculateurs de signature de transaction) pour créer le mot de passe à usage unique (OTP) final.
```

Comment:
"la saisie d'un code PIN valide, d'un déverrouillage biométrique, d'une clé USB ou d'un appairage NFC, ou d'une valeur supplémentaire" → "la saisie d'un code PIN valide, un déverrouillage biométrique, une clé USB ou un appairage NFC, ou d'une valeur supplémentaire"

i.e. remove the "de" everywhere because we are not talking about "la saisir d'une clé USB"